src/Coupon/SyncerHooks.php

```diff
@@ -194,7 +196,7 @@ public function pre_delete( int $coupon_id ) {
 	protected function handle_update_coupon( WC_Coupon $coupon ) {
 		$coupon_id = $coupon->get_id();
 
-		if ( $this->notifications_service->is_ready() ) {
+		if ( $this->notifications_service->is_ready( self::get_coupons_datatype() ) ) {
```

Comment:
Same here we should use `$this->` instead of `self::`

Reply:
Thanks. Fixed here https://github.com/woocommerce/google-listings-and-ads/commit/e595f4c28c4832b84f9f0079e3c9554aeac82124